test/builder/pipelineRun.go

```diff
@@ -135,14 +135,65 @@ func Abort() PipelineRunSpecOp {
 	}
 }
 
-// Logging creates a PipelineRunSpecOp which adds Logging to the PipelineRun
-func Logging(runID *api.CustomJSON) PipelineRunSpecOp {
+// LoggingRunID creates a PipelineRunSpecOp which adds Logging to the PipelineRun
+func LoggingRunID(runID *api.CustomJSON) PipelineRunSpecOp {
```

Comment:
To be consistent...?
```suggestion
func LoggingWithRunID(runID *api.CustomJSON) PipelineRunSpecOp {
```
Or all of them without `With`.